front/components/table/menuCell.tsx

```diff
@@ -67,16 +68,34 @@ const MenuCell: MenuCellComponent = ({
               {t('details')}
             </MenuItem>
           )}
-          {canEdit && onEdit && (
-            <MenuItem onClick={() => onEdit(itemId)} icon={<EditIcon />}>
-              {t('edit')}
-            </MenuItem>
-          )}
-          {(onDuplicate || onDestroy || onArchive || canEdit) && (
-            <MenuDivider />
+          {onEdit && (
+            <Tooltip
+              label={tooltip ? tooltip['edit'] : ''}
```

Comment:
pourquoi tu as du faire cette notation ? tooltip.edit ne jouait pas ?

Reply:
+1

Reply:
it could be empty